daemon/integration/integration_test.go

```diff
@@ -39,7 +41,7 @@ func TestRunSimpleCommand(t *testing.T) {
 	runId := assertRun("ignore", "running", "nil", statusReq[:], t)
 
 	// give the command time to finish running
-	time.Sleep(30 * time.Millisecond)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Yes, I had the TestRun2Commands test fail randomly a couple of times locally and saw some failed builds in travis due to it.  I'm just increasing the time spacing on these for now to make them very unlikely.